config/config.yaml

```diff
@@ -245,5 +246,5 @@ PgNotifier:
   retriggerCount: 500
   trackBatchInterval: 2s
   maxAttempt: 3
-EnableMultitenancy: false
-LegacyAppHandler: true
+EnableMultitenancy: true
```

Comment:
by default we will be enabling multitenancy?

Reply:
Yes, this was created to test the multi tenant stat based pickup feature. Will remove this in later PR.

Reply:
removed it.